aspnetcore/tutorials/razor-pages/sql.md

```diff
@@ -5,7 +5,7 @@ description: Part 4 of tutorial series on Razor Pages.
 ms.author: wpickett
 ms.date: 8/20/2024
 uid: tutorials/razor-pages/sql
----
```

Comment:
I'll revert this change after a VTP. It's here just to see how it looks next to the live version.